micro_ros_setup/config/zephyr/generic/client_uros_packages.repos

```diff
@@ -0,0 +1,38 @@
+repositories:
+  eProsima/Micro-CDR:
+    type: git
+    url: https://github.com/eProsima/micro-CDR.git
+    version: v1.1.2
```

Comment:
I think it could point to `dashing` tag.

Reply:
Done